embedded-hal/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "embedded-hal"
+version = "1.0.0-rc1"
+authors = ["Rust Embedded Working Group <hal@teams.rust-embedded.org>"]
+edition = "2018"
+categories = ["asynchronous", "embedded", "hardware-support", "no-std"]
+description = " A Hardware Abstraction Layer (HAL) for embedded systems "
+documentation = "https://docs.rs/embedded-hal"
+keywords = ["hal", "IO"]
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/rust-embedded/embedded-hal"
+
+[dependencies.embedded-hal-delay]
+path = "../embedded-hal-delay"
+# version = "1.0"
+
+[dependencies.embedded-hal-digital]
+path = "../embedded-hal-digital"
+# version = "1.0"
+
+[dependencies.embedded-hal-i2c]
+path = "../embedded-hal-i2c"
+# version = "1.0"
+
+[dependencies.embedded-hal-serial]
+path = "../embedded-hal-serial"
+# version = "1.0"
```

Comment:
How these crates will be updated in terms of visibility from `embedded-hal`? Will `embedded-hal` have version bump each time one of the child crates is updated?

Reply:
embedded-hal will only bump version when they want to do a release. The underlying crates can be released as often as the authors feel is necessary.

* Using embedded-hal-serial is 'unstable' (although it follows semver, things move fast)
* Using embedded-hal is 'stable' (it picks up changes from embedded-hal-XXX relatively slowly and with an eye on backwards compatibility)

Reply:
What is the intended use for these not-yet-stable releases? I believe that hal providers will use stable versions anyway.

Reply:
New HALs mainly, or proposed breaking changes. You can depend your chip crate or driver crate on the bleeding edge if necessary, or on the stable release.

Think of HEAD as nightly, a release of embedded-hal-XXX as beta, and embedded-hal as stable.

Reply:
Hmm, in this model, driver that implements the bleeding edge interface will be incompatible with the existing ecosystem without numerous compatibility layers. I don't know if such driver will be useful.